README.md

```diff
@@ -28,6 +28,21 @@ The site is published automatically by [Netlify](https://www.netlify.com/) whene
 
 We strongly encourage you to contribute to this site! For more information, see the [contributing](CONTRIBUTING.md) guide.
 
+### Publishing new Jaeger version
+
+Each Jaeger version is documented in a separate directory e.g. [content/docs/1.8](./content/docs//1.8).
+
+When submitting changes for the new version please split commits with changes
+and files copy of the previous documentation files. The following steps shows
+a possible workflow:
+
+1. copy files from old version to a new directory
+2. add the new version to [comfig.toml](./config.toml)
```

Comment:
comfig.toml -> config.toml

Reply:
Just wondering if this change should be made with the documentation changes in step 4 - so they are not actually visible until the updates have been applied?

Reply:
+1

Reply:
Actually it would not show up on the website until it's added in the config

Reply:
Yeah - so we don't want the plan copy to show under the new version - it should only show up once the copy has been updated with the new version's changes.

Reply:
My idea was to do it in one PR so there won't be a plain copy deployed on master - only on PR. Then if you deploy this commit it basically ensures that infra works. The second commit would show only the changes related in docs without docs infra.

Reply:
Then I will merge this and see how it goes.